phys/module_bl_ysu.F

```diff
@@ -254,6 +332,14 @@ subroutine ysu(u3d,v3d,th3d,t3d,qv3d,qc3d,qi3d,p3d,p3di,pi3d,               &
               ,rthraten=rthraten(ims,kms,j),p2diORG=p3di(ims,kms,j)            &
               ,ysu_topdown_pblmix=ysu_topdown_pblmix                           &
               ,ctopo=ctopo(ims,j),ctopo2=ctopo2(ims,j)                         &
+              ,a_u2d=a_u(ims,kms,j), a_v2d=a_v(ims,kms,j)                      &
+              ,a_t2d=a_t(ims,kms,j), a_q2d=a_q(ims,kms,j)                      &
+              ,b_u2d=b_u(ims,kms,j), b_v2d=b_v(ims,kms,j)                      &
+              ,b_t2d=b_t(ims,kms,j), b_q2d=b_q(ims,kms,j)                      &
+              ,b_e2d=b_e(ims,kms,j), a_e2d=a_e(ims,kms,j)                      &
+              ,sfk2d=sfk(ims,kms,j), vlk2d=vlk(ims,kms,j)                      &
+              ,dlu2d=dl_u(ims,kms,j), dlg2d=dlg(ims,kms,j)                     &
```

Comment:
@eahendricks Once you change above, these arrays will need to changed too, and those declaration inside ysu2d.